columnflow/calibration/cms/jets.py

```diff
@@ -917,6 +941,30 @@ def jer(self: Calibrator, events: ak.Array, **kwargs) -> ak.Array:
         events = set_ak_column_f32(events, f"{self.met_name}.phi_jer_up", met_phi_up)
         events = set_ak_column_f32(events, f"{self.met_name}.phi_jer_down", met_phi_down)
 
+        for name in self.jec_uncertainty_sources:
+            for junc_dir in ("up", "down"):
+
+                jets = events[jet_name]
+                jets = set_ak_column_f32(jets, "pt", jets[f"pt_jec_{name}_{junc_dir}"])
+                jets = set_ak_column_f32(jets, "mass", jets[f"mass_jec_{name}_{junc_dir}"])
+                jetsum = jets.sum(axis=1)
```

Comment:
Same as above, it would be better to make a explicit copy here or use local variables.